testing_containers/posix_ldap_multiuser/run.sh

```diff
@@ -15,4 +15,4 @@ if test "${PIP_INDEX_URL}" != ""; then
     ENVS="${ENVS} PIP_INDEX_URL=${PIP_INDEX_URL}"
 fi
 
-sudo -u agentuser -i ${ENVS} /home/agentuser/run_agent.sh
\ No newline at end of file
+sudo -u agentuser -i "${ENVS}" /home/agentuser/run_agent.sh
```

Comment:
```suggestion
sudo -u agentuser -i ${ENVS} /home/agentuser/run_agent.sh
```

`ENVS` is built-up in this script by appending multiple env var definitions together. This needs to be interpreted as multiple args to the shell, so it cannot be quoted.

Reply:
Fixed